### ISSUE TITLE
kann_class_error() is broken for KANN_C_CEB

### ISSUE THREAD

**Issue by grahamgower** [attractivechaos/kann#34]
`kann_class_error()` always returns 0 for networks with `KANN_C_CEB` cost.

**Comment by attractivechaos** [comment]
`kann_class_error()` doesn't work with binary cross-entropy `KANN_C_CEB`. With this cost, there can be multiple `1`s, but `kann_class_error()` assumes there is only one `1`.

**Comment by grahamgower** [comment]
Yep, thanks. I wasn't sure if this was intentional or not. The mnist-cnn.c example uses `KANN_C_CEB` with `kann_train_fnn1()`, which then outputs 0% class error.

**Comment by grahamgower** [comment]
Oh, wait. I'm an idiot. :) Sorry for the noise.
